src/cli/RecipeCli.java

```diff
@@ -57,6 +57,21 @@ private void saveIngredients(Integer recipeId, List<Integer> ingredientIds)
         return idList;
     }
 
+    private String getRecipeTable(List<Recipe> recipes) {
+        CliTable table = new CliTable(new String[] { "ID", "Name", "Category", "More info..." });
+        for (Recipe recipe : recipes) {
+            table.rows.add(
+                new String[] {
+                    String.valueOf(recipe.id),
+                    recipe.name,
+                    recipe.category,
+                    "Run `get` sub-command for more info",
+                }
+            );
+        }
+        return table.toString();
+    }
+
```

Comment:
I didn't explain how to use the `CliTable` functionality. I basically just copied what was in the `list` method, put it here, and called it in your search function to print out the recipe results formatted in a table.